docs/accessibility.md

```diff
@@ -0,0 +1,221 @@
+
+If you can build your layout neither with `reactstrap` nor with semantic tags, please insert `aria-role` in them. There are examples of how to use `aria-role` in: [Tables](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role), [Progress Bar](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role), [List](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/List_role).
+
+`aria-role` is important because it gives meaning to the generic HTML tag. That gives some direction to screen readers on which functionalities to provide and some information to users.
+
+**In summary**, implement your layout trying to use these in the following order:
+
+1. Use `reactstrap`;
+2. Use HTML semantic tags;
+3. Use `aria-role` attributes.
+
+### Reactstrap
+
+Some elements in Treeherder's UI are borrowed from the `reactstrap` library. As a general rule, its elements are already accessible.
+That library is built on top of Bootstrap, therefore the [Bootstrap Accessibility documentation](https://getbootstrap.com/docs/4.0/getting-started/accessibility/) is the one to look for, if in doubt.
+
+There are some issues on **Color Contrast** that are pointed by Bootstrap itself. This was solved in Treeherder by providing [**_custom color classes_**](accessibility.md#colors).
+
+### Font Awesome
+
+Treeherder also uses the `Font Awesome` library for some icons. Usually, there is no need to insert an additional explanation to them. However, if the icon has a semantic meaning, you can insert it by adding a `title` prop in the `FontAwesomeIcon` component.
+
+```jsx
+<FontAwesomeIcon icon={icon} title="Icon description" />
+```
+
+## Colors
+
+Treeherder uses [Bootstrap color](https://getbootstrap.com/docs/4.0/utilities/colors/) utility classes. While most of them work, there are a couple that needed a little tweaking to increase **color contrast**, according to [WCAG standards](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast?utm_source=devtools&utm_medium=a11y-panel-checks-color-contrast).
+
+So, we ask that you substitute the Bootstrap color to the Treeherder one, according to the following table:
+
+| Bootstrap Color | Treeherder Color       |
+| --------------: | :--------------------- |
+|      `btn-info` | `btn-darker-info`      |
+|     `text-info` | `text-darker-info`     |
+| `btn-secondary` | `btn-darker-secondary` |
+
+For example, if inserting a text color into an element, use `className="text-darker-info"`, instead of using `className="text-info"`.
+
+### Colors in buttons
+
+In [`reactstrap`'s `Button` component](https://reactstrap.github.io/components/buttons/), you can insert a `prop` specifying its color. In that case, you use only the color name, without its prefix.
```

Comment:
The subtitle isn't necessary - it can be in the "Colors" section.

I think this custom color prop is applicable to any reactstrap component that accepts a color prop, right? Please reword if that's the case. If not, that should be added in a separate pull request.

Reply:
> I think this custom color is applicable to any reactstrap component that accepts a color prop, right? Please reword if that's the case. If not, that should be changed in a separate pull request.

This props can be inserted, but it won't work on all of them. I'll both reword and send a separate PR.